meta/saiserialize.cpp

```diff
@@ -1523,6 +1551,10 @@ std::string sai_serialize_attr_value(
 {
     SWSS_LOG_ENTER();
 
+    if(meta.attrvaluetype == SAI_ATTR_VALUE_TYPE_SEGMENT_LIST)
+    {
+      SWSS_LOG_ERROR("serialize: Atrribute value is segment list");
+    }
```

Comment:
throw in this case ? alos add space after "if", and 4 spaces in any semgment you have between { }, now you have 2